tools/eksDistroBuildToolingOpsTools/pkg/repoManager/repoManager_test.go

```diff
@@ -0,0 +1,89 @@
+package repoManager_test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	gogithub "github.com/google/go-github/v48/github"
+
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/github"
+	githubMocks "github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/github/mocks"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/repoManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/retrier"
+)
+
+const (
+	HttpOkStatusCode = 200
+)
+
+var (
+	TestRepoOwner      = "TestTestPerson"
+	TestRepo           = "TestRepo"
+	TestFilePath       = "test/path/to/file"
+	TestFileContent    = "Test content: Hello there"
+	FileName           = "File"
+	IssueBody          = "Body"
+	IssueAssignee      = "Jeff"
+	IssueState         = "Open"
+	ReturnIssueHtmlUrl = "https://github.com/testrepo/issues/999"
+	IssueNumber        = 999
+)
+
+func expectedLabels() *[]string {
+	return &[]string{"sup", "test"}
+}
+
+func TestRepoManagerGetFileSuccess(t *testing.T) {
```

Comment:
let's also add a negative test-case for the rate-limiting fail; check out https://github.com/aws/eks-distro-internal/blob/main/tools/vulnerability-processor/pkg/issueManager/issueManager_test.go#L70 as an example